frontend/src/js/components/PageViewer/PageCache.ts

```diff
@@ -119,6 +130,40 @@ export class PageCache {
     };
   };
 
+  refreshPreview = (pageNumber: number, preview: Promise<CachedPreview>, containerSize: number): CachedPage => {  
```

Comment:
👌 This is a much nicer approach than the re-requesting of the preview that we ended up with on Friday!

Is there a way - in the name of the function, or a comment - to make it clear that this does *not* trigger a new request, but only a re-render? Because it shares the "refresh" name with `getPageAndRefreshHighlights`, which does trigger a new request.

Reply:
Thanks Jo, I changed it to be reRenderPagePreview, is it good?

Reply:
Great!